pkg/internal/apis/certmanager/validation/issuer.go

```diff
@@ -35,7 +35,19 @@ import (
 
 func ValidateIssuer(obj runtime.Object) field.ErrorList {
 	iss := obj.(*v1alpha1.Issuer)
-	allErrs := ValidateIssuerSpec(&iss.Spec, field.NewPath("spec"))
+	fldPath := field.NewPath("spec")
+	allErrs := ValidateIssuerSpec(&iss.Spec, fldPath)
+
+	for _, solver := range iss.Spec.IssuerConfig.ACME.Solvers {
+		s := solver.HTTP01
+		if s != nil &&
+			s.Ingress != nil &&
+			s.Ingress.PodTemplate != nil &&
+			0 != len(s.Ingress.PodTemplate.Namespace) {
```

Comment:
We are quite keen to not over scope the `PodTemplate` field - the pod isn't the only resource that will be created in this namespace (order, challenge, ingresses...). Can we move this to a new field on the HTTP01 solver resource?